Formula/gphoto2.rb

```diff
@@ -22,4 +22,8 @@ def install
                           "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    `gphoto2 -v`
```

Comment:
Minor note for future, we'll accept tests like this if you can't think of anything more extensive or more extensive isn't possible but they should match [this format](https://github.com/Homebrew/homebrew-core/commit/990a3b799079a3afd55cb48fc54b9abc0a436ec7#diff-dd99875eb6cc0ca46b8f78e1df719b45R27) rather than as written here. I fixed it in post this time.